### DISCUSSION TITLE
aspect ratio class doesn't work

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#8045] [Help] [answered]
I tried the aspect ratio classes on an image but it didn't work:
https://play.tailwindcss.com/WWK7yZ9Qme

**Comment by CFarhad** [comment, marked as answer]
hi 
use the `aspect-square` in image classes ([demo](https://play.tailwindcss.com/q803sNhT9z))

**Reply by adamwathan** [reply]
Yep +1, it works fine you just need to put it on the right element. In your demo the `div` is square but the image is overflowing it.

Can be fixed by making sure the image fills the shape of the parent div (https://play.tailwindcss.com/pesKlqnMWo) but the right solution is to put the aspect-ratio utility directly on the image.
